packages/components/src/table/index.js

```diff
@@ -179,8 +179,13 @@ class TableCard extends Component {
 		}
 	}
 
-	onSearch( values ) {
-		const { compareParam, searchBy, baseSearchQuery } = this.props;
+	onSearchChange( values ) {
```

Comment:
Thanks for taking care here and making sure to not leak tracks logic into the npm packages 👍